src/main/java/com/minecolonies/rankup/internal/command/RankupSubcommand.java

```diff
@@ -43,12 +40,12 @@ protected final Rankup getPlugin()
         return empty;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Return a copy of "empty". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS2384)